marimbabot_audio/src/sequence_evaluation.py

```diff
@@ -0,0 +1,161 @@
+import rospy
+from marimbabot_msgs.msg import HitSequence as HitSequenceMsg
+from marimbabot_msgs.msg import SequenceMatchResult as SequenceMatchResultMsg
+from marimbabot_msgs.msg import NoteOnset as NoteOnsetMsg
+
+class Comparison():
```

Comment:
Would it make sense to compare them with dynamic time warping as an extra metric, to filter out constant offsets. Because it would be preferable, in many scenarios, to keep playing with accurate relative timings instead of playing faster to match the original speed again after we missed one timing.

Reply:
how about talking about the details in person, I am not sure what you want to do exactly. And why do you need to compare it with relative time? I will be in the student lab every day.

Reply:
If you compare the absolute timings and the robot is too slow at one note it carries the offset and gets penalized for all subsequent notes. With relative timings you only compare the timing from one note to the other meaning it only gets punished for the one wrong note. 

I don't know if this chart helps. Hits are denoted with `|` and misses are marked with `x`. Each character is a timestamp.
```
Target:      |---|---|----|
Measurement: |----|---|----|
Absolute:    |----x---x----x
Relative:    |----x---|----|
```